about.html

```diff
@@ -0,0 +1,131 @@
+          <nav class="main-nav">
+              <div class="icon-nav">
+                  <a href="index.html"><img src="./images/mylogo.png" alt="logo image"></a>
+              </div>
+              <div class="icon-menu-mobile">
+                  <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+              </div>
+              <ul class="bottom-nav">
+                  <li><a  class="active" href="about.html">About</a>
+                  <li><a href="#program">Services</a></li>
+                  <li><a href="#about-partners">Partners</a></li>
+                  <li><a href="#Testimonials">Testimonials</a></li>
+                  <li><a href="#">Blog</a></li>
+                  <li><button type="button">Follow US</button></li>
+              </ul>
+          </nav>
+      </div>
+  </div>
+
+  <div class="nav-mobile">
+      <div class="cross">
+          <img src="images/icons/cross.svg" alt="cross">
+      </div>
+      <ul class="container">
+          <li><a href="about.html">About</a>
+          <li><a href="#program">Services</a></li>
+          <li><a href="#testimonials">Testimonials</a></li>
+          <li><a href="#about-partners">Partners</a></li>
+          <li><a href="#">Blog</a></li>
+      </ul>    
+  </div>
+  </header>
+
+  <main> 
+    <section id="about-container1">
+      <p class="about-description">This is Lugard's Tech Development Academy '3ple Tech'</p>
+      <hr class="about-description-hr">
+      <h1 class="about-header">AT 3PLE TECH OUR DEDICATION IS BASED ON BEST PRACTICES AND CLIENTS SATISFACTION.</h1>
+      <div id="about-theme">
+        <p class="about-theme">
+          3ple Tech is service delivery and learning based academy, with a very 
+          vast network of resource persons and professional developers. Our team comprises of dedicated
+          workers and effective code developers. Our objective is to bridge knowledge-based gap of the tech 
+         industry especially from the African perspective.
+        </p>
+      </div>
+      <p class="about-contact">
+        Please contact us via email or phone call for further enquiry to 3ple Tech 
+        and service delivery.
+      </p>
+      <p class="about-contact-mail">agulugard@gmail.com</p>
+      <hr class="about-contact-hr">
+    </section>  
+  
+    <div class="about-logo">
+      <div class="about-logo2">
+        <img src="./images/mylogo.png" class="about-mylogo" alt="mylogo">
+        <p class="academy-description">3ple Tech academy<br>October 2022<br>Enugu, Biafra.</p>
+      </div>
+    </div>
+    <div id="about-members">
+      <h2>Team members</h2>
+      <p class="about-members-description">Take a look at some of the 3ple Tech academy software Developers and instructors</p>
+      <ul>
+        <li><img src="images/about_2013.png" class="about-pic" alt="illustrative-pic"></li>
+        <li><img src="images/about_2011.png" class="about-pic" alt="illustrative-pic"></li>
+      </ul>
```

Comment:
Great job adding the images as requested by the previous reviewer however, note that in desktop version the images are arranged side by side and in desktop top bottom. For both you have the images arranged top to bottom
This is the design 
<img width="1062" alt="Screenshot 2022-10-29 at 18 54 06" src="https://user-images.githubusercontent.com/48546992/198841589-afff18a0-a8b9-495e-b34e-772a87820022.png">
This is your desktop design
<img width="1667" alt="Screenshot 2022-10-29 at 19 07 12" src="https://user-images.githubusercontent.com/48546992/198841636-4b29b589-a386-43f3-9ead-79cf45d02e22.png">